eunji/src/components/Comment.tsx

```diff
@@ -0,0 +1,80 @@
+import { ChangeEvent, FormEvent, useState } from "react";
+import styled from "styled-components";
+
+interface IComment {
+    content: string,
+}
+
+export default function Comment() {
+    const [Comments, setComment] = useState<IComment[]>([{ content: '' }]);
```

Comment:
```suggestion
    const [comments, setComment] = useState<IComment[]>([{ content: '' }]);
```

소문자로 시작하는 게 더 관례적일 거 같아요~

Reply:
피드백 반영해서 푸시했습니다!